app/app.go

```diff
@@ -174,6 +175,8 @@ var (
 		govtypes.ModuleName:            {authtypes.Burner},
 		ibctransfertypes.ModuleName:    {authtypes.Minter, authtypes.Burner},
 	}
+
+	Upgrades = []upgrades.Upgrade{v2_6.Upgrade}
```

Comment:
why we need the list of upgrades

Reply:
it's the standard in the industry. It provide flexibility in case we need to update through a lot of upgrades at once